src/bin/vip-wp.js

```diff
@@ -252,13 +252,40 @@ const bindReconnectEvents = ( {
 	} );
 };
 
+// Command examples
+const examples = [
+	{
+		usage: `vip @example-app.develop -- wp site list`,
+		description:
+			'Retrieve the list of network sites on the develop environment of the "example-app" WordPress multisite application.',
+	},
+	{
+		usage: `vip @example-app.production --yes -- wp user list`,
+		description:
+			'Retrieve the list of Super Admins on the production environment and automatically answer "yes" to the confirmation prompt.',
+	},
+	{
+		usage: `vip @example-app.develop -- wp post list --posts_per_page=100 --url=site.example.com`,
+		description: 'Retrieve the list of posts for the network site site.example.com.',
+	},
+	{
+		usage:
+			`vip @example-app.develop --yes -- wp\n` +
+			`    - example-app.develop:~$ wp option get home\n` +
+			`    - https://dev.example.com`,
+		description:
+			'Enter interactive shell mode to run a WP-CLI command like a standard terminal or SSH session.',
```

Comment:
```suggestion
			'Use the VIP-CLI to launch an interactive WP-CLI console to run WP-CLI commands on the develop environment of the "example-app" application.',
```

This shell mode can only run WP-CLI commands. Since standard terminal commands are not available, and this is not an SSH session, we should avoid making that correlation in the description.

We should avoid the term `shell` here to avoid confusion about how this is also different from `vip @example-app.develop -- wp shell` - which may be worth including in a separate usage example.

Reply:
[Related discussion in Slack](https://a8c.slack.com/archives/C245KEKF1/p1729092958288579). 

The WP-CLI command `wp shell` is disallowed on the VIP Platform:
https://docs.wpvip.com/vip-cli/wp-cli-with-vip-cli/disallowed-commands/#:~:text=server%3A%20*-,shell,-%3A%20*